src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +21,26 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws FileNotFoundException {
+        TreeSet<Character> treeSet = new TreeSet<>();
+        try (FileReader file = new FileReader(fileName)) {
```

Comment:
```suggestion
        try (FileReader reader= new FileReader(fileName)) {
```
це не файл, а об'єкт, який його зчитує, тому називати його треба відповідно